app-admin/keepassxc/keepassxc-2.7.9.ebuild

```diff
@@ -28,7 +28,7 @@ fi
 
 LICENSE="LGPL-2.1 GPL-2 GPL-3"
 SLOT="0"
-IUSE="X autotype browser doc keeshare +network test yubikey"
+IUSE="X autotype browser doc keeshare +network test yubikey ssh-agent keyring"
```

Comment:
flags are not sorted alphabetically here

also please add them by default: +ssh-agent +keyring